src/Generator.php

```diff
@@ -50,9 +50,9 @@ public function __construct(
         // Find the drivers to add to the extra/interfaces
         $this->detectDrivers();
 
-        $this->extra = array_merge($this->extra, $this->config->get('ide-helper.extra'));
-        $this->magic = array_merge($this->magic, $this->config->get('ide-helper.magic'));
-        $this->interfaces = array_merge($this->interfaces, $this->config->get('ide-helper.interfaces'));
+        $this->extra = array_merge((array)$this->extra, (array)$this->config->get('ide-helper.extra'));
+        $this->magic = array_merge((array)$this->magic, (array)$this->config->get('ide-helper.magic'));
+        $this->interfaces = array_merge((array)$this->interfaces, (array)$this->config->get('ide-helper.interfaces'));
```

Comment:
> all I know is that one of them clearly was null 

The reason is likely that your config didn't have these settings at all. Maybe because you upgraded from an older version.

However, the correct solution is not to cast but to make the config lookups to default to an empty array.

Would you mind updating the PR?